test/edu-benefits/1990/components/SubmitMessage.unit.spec.jsx

```diff
@@ -5,7 +5,7 @@ import { expect } from 'chai';
 import SubmitMessage from '../../../../src/js/edu-benefits/1990/components/SubmitMessage';
 import { createVeteran } from '../../../../src/js/edu-benefits/1990/utils/veteran';
 
-describe('<BenefitsWaiverFields>', () => {
+describe('<SubmitMessage>', () => {
```

Comment:
This seems like an odd change for this PR. What prompted it? Something with skin-deep 1.0, or should this have been `<SubmitMessage>` already?